### ISSUE TITLE
Add news alerts for watched stocks

### ISSUE THREAD

**Issue by TNowalk** [TNowalk/stock-slack-bot#15]
When poll a watched stock, also poll the company's news feed.  The format is as follows: http://finance.yahoo.com/rss/headline?s=[csv of symbols]

May have to store the GUID locally to keep track of what news items have been displayed.  If on a poll a new story was published, notify the user watching that stock and post a link.  When a user adds a stock to the watch list, pull down the news feed and mark all as "seen" so the user isn't immediately blasted with notifications.

Will have to keep track of what users have been notified about what stories so if two people are watching the same stock they will both get notified if a story gets updated.

Blocked by #6
